.github/workflows/publish.yml

```
@@ -65,6 +65,4 @@ jobs:
       - name: Install dependencies
         run: npm clean-install
       - name: Publish to npm
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: npm publish --provenance || true
+        run: npm publish
```

This should have been

```suggestion
        run: npm publish || true
```

---

Resolved with #274